sympy/core/power.py

```diff
@@ -135,7 +135,10 @@ class Pow(Expr):
     +--------------+---------+-----------------------------------------------+
     | 1**oo        | nan     | Because there are various cases where         |
     | 1**-oo       |         | lim(x(t),t)=1, lim(y(t),t)=oo (or -oo),       |
-    | 1**zoo       |         | but lim( x(t)**y(t), t) != 1.  See [3].       |
+    |              |         | but lim( x(t)**y(t), t) != 1.  See [3].       |
+    +--------------+---------+-----------------------------------------------+
+    | b**zoo       | nan     | Because b**z has no limit as |z|->oo when z   |
```

Comment:
`z -> zoo` might behave better.

Reply:
It turned out that |z| has a [special meaning](http://www.sphinx-doc.org/en/stable/rest.html#substitutions) in Sphinx, the software building HTML pages from these docstrings. This is why you got this error. It doesn't come up when you work locally, because you don't build HTML documentation locally.

Reply:
So, as jksuom suggested, replacing "|z|->oo when z is a complex number" with "z->zoo" will avoid this issue (and be shorter, which is also good for this table)